src/test/java/practice/DemoSite_Register.java

```diff
@@ -32,6 +33,7 @@ public void register() {
 		// (Jenny)5 - fill up phone number : "9198805671"
 
 		// (Danny)6 - select gender : male
+		driver.BaseClass.getDriver().findElement(By.xpath("//*[@id=\"basicBootstrapForm\"]/div[5]/div/label[1]/input")).click();
```

Comment:
Your code will work here. But , please store webElement in page class. We using POM and OOPs concept. After that, you can either setup a method to click on male radio button or use the webElement variable on test class and just click it.